bst_plugin/forward/nst_headmodel_get_gains.m

```diff
@@ -1,28 +1,43 @@
-function gains = nst_headmodel_get_gains(nirs_head_model,iwl, channel_def, selected_chans)
+function gains = nst_headmodel_get_gains(head_model, unused_iWL, sChannel, selected_channels)
```

Comment:
Should i keep the previous functrion definition or  remove the unused parameter ? I think keeping it is nice for backward compatibility